.github/workflows/gh-actions-release.yml

```diff
@@ -3,32 +3,24 @@ name: 'GH Actions - Release'
 on:
   release:
     types:
-      - published
       - created
       - edited
-      - prereleased
```

Comment:
We'll have to learn more about the event types; but I suspect we'll need those still. What we might want to do is have some kind of check to see if a target is already there before uploading unless a special flag is set to overwrite the target.